### PR TITLE
Move SendTag pages under account route

### PR DESCRIPTION
**Pull request by youngkidwarrior**
M

### CODE REVIEW

**Comment by youngkidwarrior** [comment]
* **#200** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/200?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `dev`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @youngkidwarrior and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/app/features/account/sendtag/screen.tsx
Comment (0xBigBoss) [inline packages/app/features/account/sendtag/screen.tsx:1]: if you could, add a playwright spec for this page. It should use the base tests from one of the profile.onboarding.spec.ts

https://github.com/0xsend/sendapp/blob/dev/packages/playwright/tests/profile.onboarded.spec.ts#L1

possible, rename the [checkout spec](https://github.com/0xsend/sendapp/blob/dev/packages/playwright/tests/checkout.logged-in.spec.ts#L1) to jsut be account-sendtag-checkout.logged-in.spec.ts.

**Review by 0xBigBoss — APPROVED** [review]
nice, let's see if this breaks any tests, if you could add more test coverage pls 🙏

File: packages/app/features/account/sendtag/screen.tsx
Comment (youngkidwarrior) [reply packages/app/features/account/sendtag/screen.tsx:1]: let me know if this works. tests are passing

Comment (0xBigBoss) [reply packages/app/features/account/sendtag/screen.tsx:1]: looks good. https://github.com/0xsend/sendapp/actions/runs/8312676705/job/22747836963

are you able to also add a new jest unit test for the account/sendtag screen? We have a lot of screens now that don't have any snapshot tests. Should be pretty easy, looks like we only need to mock the `useUser` hook.

**Review by 0xBigBoss — APPROVED** [review]

File: packages/app/features/account/sendtag/screen.tsx
Comment (youngkidwarrior) [reply packages/app/features/account/sendtag/screen.tsx:1]: Sure, let's merge this so QA can look at it. Ill add the test in the next PR

**Comment by youngkidwarrior** [comment]
### Merge activity

* **Mar 17, 3:13 PM EDT**: @youngkidwarrior started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/0xsend/sendapp/200).

**Pull request merged**